ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -22,17 +35,42 @@
     state: true
     persistent: true
 
-- name: Install Crane configuration file
-  copy:
-    src: crane.conf
-    dest: /etc/crane.conf
-    owner: root
-    group: root
-    mode: 0644
+- block:
+  - name: Install Crane configuration file
+    copy:
+      src: crane.conf
+      dest: /etc/crane.conf
+      owner: root
+      group: root
+      mode: 0644
+
+  - name: Open firewall port for Crane (firewalld)
+    firewalld:
+      port: "5000/tcp"
+      state: enabled
+      permanent: true
+      immediate: true
+  when:
+    - not (ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 6)
```

Comment:
I would normalize conditional blocks to use the same logic. For example on line 17 it is checking if the major_version is greater than here it is not equal. That said I would normalize and use greater than approach or not equal on both places.

Also, if choosing the not equal approach maybe `ansible_distribution == "RedHat" and ansible_distribution_major_version|int != 6` is more readable.